codec-http/pom.xml

```diff
@@ -67,6 +67,30 @@
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+    </dependency>
```

Comment:
This should have `test` scope

Reply:
Done is parent's dependencyManagement block: https://github.com/netty/netty/pull/10960/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R763-R768